builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -136,14 +137,25 @@ node() {
 			}
 		}
 
-		def requestId = utilModule.generateRequestId()
-		if (requestId != null) {
-			events.setRequestId(requestId)
-			environmentDeploymentMetadata.setRequestId(requestId)
+		def requestId 
+		if(request_id == 'none') {
+			requestId = utilModule.generateRequestId()
+			if (requestId != null) {
+				events.setRequestId(requestId)
+				environmentDeploymentMetadata.setRequestId(requestId)
+			} else {
+				error "Request Id Generation failed"
+			}
+		} else {
+			echo "Setting up request_id"
+			requestId = request_id
+    	}
+
+		if(request_id == 'none') {
```

Comment:
This can be included in the above condition.

Reply:
done